.vscode/settings.json

```diff
@@ -5,7 +5,6 @@
 // - Mac: $HOME/Library/Application Support/Code/User/settings.json
 {
   "tslint.enable": true,
-  "editor.formatOnSave": true,
```

Comment:
This change should be reverted, we added `editor.formatOnSave` be `true` for collaborators to enforce formatting before submitting pull-requests.